src/Peachpie.Library/Hash.cs

```diff
@@ -3024,13 +3032,422 @@ public static string crypt(string str, string salt/* mandatory since 5.6 */)
             else
             {
                 // DES
-                throw new NotImplementedException();
+                return DES.Crypt(str, salt);
             }
 
             // failure
             return salt.StartsWith("*0") ? "*1" : "*0";
         }
 
+        private static string ConvertByteArrayToString(byte[] buffer)
```

Comment:
isn't it just 
```c#
Encoding.ASCII.GetString(buffer)
```
?

Reply:
Resolved https://github.com/peachpiecompiler/peachpie/pull/593/commits/6f1865f58f6166fda85c7352d98cab2d049e5ddf